site/src/index.css

```diff
@@ -0,0 +1,41 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+	:root {
+		--content-primary: #27272a;
+		--content-secondary: #52525b;
+		--content-link: #2563eb;
+		--content-invert: #fafafa;
+		--content-disabled: #a1a1aa;
+		--content-success: #15803d;
+		--content-danger: #ef4444;
+		--surface-primary: #fafafa;
+		--surface-secondary: #f4f4f5;
+		--surface-tertiary: #e4e4e7;
+		--surface-invert-primary: #27272a;
+		--surface-invert-secondary: #3f3f46;
+		--surface-error: #450a0a;
+		--border-default: #e4e4e7;
+		--border-error: #ef4444;
+		--radius: 0.5rem;
```

Comment:
`--radius` is used in the tailwind config here,
 ```			borderRadius: {
				lg: "var(--radius)",
				md: "calc(var(--radius) - 2px)",
				sm: "calc(var(--radius) - 4px)",
			},```
			
These were both added by the shadcn installation. I was keeping this for now because Im not sure yet if other shadcn components will be expecting this. We will probably be custom styling every component anyway so this may not be necessary.